### DISCUSSION TITLE
Teedy (sismics/docs) to Paperless Migration Script

### DISCUSSION THREAD

**Discussion by shamoon** [jonaswinkler/paperless-ng#193] [General]
Figured I'd share in case anyone ever needs. Its super basic (at the moment!), just exports all documents with tags but this was the last step I needed to fully jump in to Paperless-ng 😃

Script assumes username/pw are the same, intentionally disables some security checks, etc. YMMV!

[https://gist.github.com/shamoon/496dcf859f57353113a999305dc66248](https://gist.github.com/shamoon/496dcf859f57353113a999305dc66248)

**Comment by jonaswinkler** [comment]
We can put this in some scripts / import / from-teedy.sh file, if you like.

**Comment by shamoon** [comment]
I didn't suggest that in the first place because it just seemed like yet-another-thing-to-maintain but its totally up to you Jonas. Honestly though Im not sure how much interest there is / will be in it?

**Comment by jonaswinkler** [comment]
Right, good point.
